### PR TITLE
Update table that compares sysbox and other runtimes. [skip ci]

### PR DESCRIPTION
**Pull request by ctalledo**
Increase isolation rating for gVisor, and adjust
the performance rating for Kata and KubeVirt.

Adjustments done per feedback in this HN thread:
https://news.ycombinator.com/item?id=29967138

Signed-off-by: Cesar Talledo <ctalledo@nestybox.com>

### CODE REVIEW

**Review by rodnymolina — APPROVED** [review]
I'm not convinced about those 4 starts for Gvisor on the "efficiency" front, especially since we've heard several complaints about Gvisor's performance penalty, but don't have cycles to look into this right now.

File: README.md [hunk 1]
```diff
@@ -27,10 +27,15 @@ two key ways:
...
-*   **Enables containers to act as VMs**:
+*   **Enables containers to run same workloads as VMs**:
 
-    *   With Sysbox, containers can run systemd, Docker, Kubernetes, K3s, buildx,
-        legacy apps, and more seamlessly & securely.
+    *   With Sysbox, containers can run system-level software such as systemd,
+        Docker, Kubernetes, K3s, buildx, legacy apps, and more seamlessly &
+        securely.
+
+    *   These software can run inside Sysbox containers without modification and
```
Comment (rodnymolina) [inline README.md:36]: s/these/this

File: README.md [hunk 2]
```diff
@@ -148,10 +155,21 @@ More on how to use Sysbox [here](#using-sysbox).
...
+As shown, Sysbox enables unprivileged containers to run system-level workloads
+such as systemd, Docker, Kubernetes, etc., seamlessly, while giving you a
+balanced approach between container isolation, performance, efficiency, and
+portability.
+
+And it does this with minimal configuration changes to your existing infra: just
+install Sysbox and configure your container manager/orchestrator to launch
+containers with it, using the image of your choice.
+
+Note that while Sysbox hardens isolation of standard containers and voids the
```
Comment (rodnymolina) [inline README.md:167]: i/hardens/hardens the/

File: README.md [hunk 3]
```diff
@@ -42,9 +47,10 @@ that have hardened isolation and can run almost any workload that runs in VMs.
 Sysbox does this by making the container resemble a VM-like environment as much
 as possible, using advanced OS virtualization techniques.
 
-Unlike alternative runtimes such as Kata and KubeVirt, **it does not use VMs**;
+Unlike alternative runtimes such as Kata and KubeVirt, **it does not use VMs**.
```
Comment (rodnymolina) [inline README.md:50]: s/this/This

File: README.md [hunk 4]
```diff
@@ -42,9 +47,10 @@ that have hardened isolation and can run almost any workload that runs in VMs.
...
 this makes it easier to use (particularly in cloud environments by avoiding
-nested virtualization). See [here](#comparison-to-related-technologies) for a
+nested virtualization), although it does not provide the level of isolation
+that VM-based runtimes do. See [here](#comparison-to-related-technologies) for a
```
Comment (rodnymolina) [inline README.md:53]: s/does not provide/does not (yet) provide/ :-)

**Review by rodnymolina — APPROVED** [review]
LG

One more thing. Add "RKE and RKE2" to the line holding "K3s and K0s" items in the comparison table.

File: README.md (hunk 1 above)
Comment (ctalledo) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (ctalledo) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (ctalledo) [reply README.md:50]: Done.

File: README.md (hunk 4 above)
Comment (ctalledo) [reply README.md:53]: Done.

**Pull request merged**